ci-operator/config/openshift-knative/kn-plugin-func/openshift-knative-kn-plugin-func-release-next.yaml

```diff
@@ -2,14 +2,14 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.16-openshift-4.9
+    tag: rhel-8-golang-1.17-openshift-4.11
 releases:
   latest:
     candidate:
       architecture: amd64
       product: ocp
       stream: nightly
-      version: "4.8"
+      version: "4.11"
```

Comment:
IMO this whole section `releases:` can be removed. It doesn't enable any additional functionality, because only unit tests are defined later. It isn't a problem for this PR, more FYI.

For E2E setup a cluster claim might be more interesting and quicker, e.g. from [S-O operator](https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.10.yaml#L203-L209).